syntax = "proto3";

message Features {
  bytes no2_data = 1;
  bytes pm10_data = 2;
  bytes pm25_data = 3;
}

message Predictions {
  bytes results = 1;
}

service Prediction {
  rpc predict(Features) returns(Predictions);
}